pontoon/homepage/models.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Homepage(models.Model):
+    text = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ('-created_at', )
```

Comment:
Is that really necessary? Default ordering is by ID, which should result in the same but reverse order as if ordering by `auto_now_add=True` fields.

Reply:
I agree, this is not useful in the current state, so let's rely on the database ordering by ID by default. Less strain on the database is always good. :-)